File: Astronomy-Hybrid/Astronomy-Hybrid/ImageCollectionViewCell.m
//
//  ImageCollectionViewCell.m
//  Astronomy-Hybrid
//
//  Created by Bobby Keffury on 2/12/20.
//  Copyright © 2020 Bobby Keffury. All rights reserved.
//

#import "ImageCollectionViewCell.h"

@interface ImageCollectionViewCell ()

@property (weak, nonatomic) IBOutlet UIImageView *imageView;

@end

@implementation ImageCollectionViewCell

- (void) prepareForReuse
{
    [super prepareForReuse];
    
    _imageView.image = [UIImage imageNamed:@"MarsPlaceholder"];
}

@end

File: Astronomy-Hybrid/Astronomy-Hybrid/ImageCollectionViewCell.swift
//
//  ImageCollectionViewCell.swift
//  Astronomy-Hybrid
//
//  Created by Vici Shaweddy on 2/12/20.
//  Copyright © 2020 Vici Shaweddy. All rights reserved.
//

import UIKit

class ImageCollectionViewCell: UICollectionViewCell {
    
    @IBOutlet weak var imageView: UIImageView!
    
    override func awakeFromNib() {
        super.awakeFromNib()

        self.imageView.image = #imageLiteral(resourceName: "MarsPlaceholder")
        self.imageView.contentMode = .scaleAspectFill
        self.imageView.clipsToBounds = true
    }
}
